specification/quantum/Quantum.Workspace/operations/jobs.tsp

```diff
@@ -0,0 +1,142 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+import "./operations.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+
+using TypeSpec.Http;
+using OpenAPI;
+
+@doc("The type of the job.")
+enum JobType {
+  @doc("Unknown job type.")
+  Unknown,
+
+  @doc("Quantum Computing job type.")
+  QuantumComputing,
+
+  @doc("Optimization job type.")
+  Optimization,
+}
+
+@doc("The status of the job.")
+enum JobStatus {
+  @doc("The job is waiting in the queue to be executed.")
+  Waiting,
+
+  @doc("The job is being executed.")
+  Executing,
+
+  @doc("The job completed with success.")
+  Succeeded,
+
+  @doc("The job completed with failure.")
+  Failed,
+
+  @doc("The job was cancelled.")
+  Cancelled,
+}
+
+@doc("Quantum computing data.")
+model QuantumComputingData {
+  @doc("The number of quantum computing items in the job.")
+  @visibility("read")
+  count: int64;
+}
+
+@doc("A job to be run in the workspace.")
+// Notes:
+// - The generated Swagger is not automatically adding the x-ms-discriminator-value
```

Comment:
Check with @allenjzhang why the discriminator value is not being added automatically.

Reply:
Discussed in meeting.

Reply:
Allen and I didn't have time to go thought this comment, but we agreed that it's a small detail that can be addressed in the future.